tests/test_overrides.py

```diff
@@ -147,3 +149,61 @@ def test_list_all(tmp_path: pathlib.Path):
     )
 
     assert expected == packages
+
+
+@patch("fromager.overrides.find_override_method")
+def test_invoke_override_with_exact_args(find_override_method: typing.Callable):
+    def default_foo():
+        return False
+
+    def foo(arg1, arg2):
+        return arg1 is not None and arg2 is not None
+
+    find_override_method.return_value = foo
+
+    assert overrides.invoke(foo, arg1="value1", arg2="value2")
```

Comment:
You're just calling invoke, so mocking the find method shouldn't be needed? Maybe you meant to call through both code paths?

Reply:
Ah yes forgot to remove that